Build/GameSettings.h

```diff
@@ -73,11 +73,10 @@ struct GAME_SETTINGS
 //Enums for the difficulty levels
 enum
 {
-	DIF_LEVEL_ZERO,
-	DIF_LEVEL_EASY,
-	DIF_LEVEL_MEDIUM,
-	DIF_LEVEL_HARD,
-	DIF_LEVEL_FOUR,
+	DIF_LEVEL_EASY = 1,
```

Comment:
It's for backward compatibility. These values are being assigned to `ubDifficultyLevel ` which is then used in couple of computations and array accesses that assume 1-based difficulty. For example
`curr->ubNumCreatures = (UINT8)(2 + gGameOptions.ubDifficultyLevel*2 + Random( 2 ))`. So I decided to stay conservative instead of risking "off by one" bugs.